### PR TITLE
script for installing PCMD on OSX for CMD

### PR DESCRIPTION
**Pull request by Levi-Leah**

### CODE REVIEW

**Comment by Levi-Leah** [comment]
@adahms 
I put the steps into the script and updated the README file. Please let me know if it works on your side, as I don't have access to OSX

File: osx-cmd-intallation.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    # Install Homebrew
+    echo 'Installing Homebrew...'
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+else
+    :
+fi
+
+echo 'Installing package dependencies...'
+
+brew Installing python3 ruby subversion
```
Comment (adahms) [inline osx-cmd-intallation.sh:29]: s/Installing/install/

File: osx-cmd-intallation.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+echo 'Installing ruby gem dependencies...'
+
+gem Installing asciidoctor concurrent-ruby haml tilt
```
Comment (adahms) [inline osx-cmd-intallation.sh:33]: Add 'sudo' to the start

s/Installing/install/

File: osx-cmd-intallation.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+else
+    :
+fi
+
+echo 'Installing package dependencies...'
+
+brew Installing python3 ruby subversion
+
+echo 'Installing ruby gem dependencies...'
```
Comment (adahms) [inline osx-cmd-intallation.sh:31]: Somewhere around here, we also need to run 'pip3 install pygit2'

**Comment by Levi-Leah** [comment]
@adahms I made the changes, should work now

**Comment by adahms** [comment]
Thanks @Levi-Leah - merging!

**Pull request merged**